l3kernel/doc/source3body.tex

```diff
@@ -335,6 +335,14 @@ \section{Naming functions and variables}
     lists (with access at both ends) and stacks.
 \end{description}
 
+\subsection{Behavior of \texttt{c}-type expansion for undefined arguments}
+
+When \texttt{c}-type expansion is applied to with and argument that is
+undefined, the result is a token equal to |\scan_stop:| (|\relax|). This will
+likely lead to low-level errors if applied to variables. As such, the
+programmer should ensure that \texttt{c}-type expansion is only applied where
+resulting variable names will definitely exist.
```

Comment:
definitely -> already ?

Reply:
No "definitely" is correct, imho. Think of `\tl_new:c`.

Reply:
that's a case I haven't considered, but "definitely" doesn't  give the right vibe to me. I think it would be better to explicitly spell it out

 ...  already exists or is being defined by the command, e.g., \tl_new:c

with better wording of course.

Reply:
Try this: keep "definitely" and add to the end of the sentence further explanation, such as:

: i.e., each of the names either already exists, or gets defined by using this command. 
. . . not exactly correct wording! 

I did not understand the "e.g., \tl_new:c" so left it out.  What is it an example of?

Reply:
@car222222 `\tl_new:c` is a command that must not be used with an already existing macro, so in the no-error-case is guaranteed to let the control sequence to `\relax`, but it'll definitely be defined by it.

I know I'm now late to suggest any changes here, but I'd argue a simple "eventually" at the end of that sentence would explain it quite elegantly in a single word.

Reply:
Re: \tl_new:c is a command that must not be used with an already existing macro

OK, but there is still a need to explain why this example is bring used, and what it explains.  None of this is at call ,clear at present, is it?